todo/src/actions/todoActions.js

```diff
@@ -0,0 +1,25 @@
+export const ADDTODO = "ADDTODO";
+export const REMOVETODO = "REMOVETODO";
+export const TOGGLETODO = "TOGGLETODO";
+
+
+export function addTodo(todo) {
```

Comment:
Since you need an `id` to reference the todos for `removeTodo` and `toggleTodo`, you can add that in your `addTodo` function:

```
let id = 0

export function addTodo(todo) {
    return {
        type: ADDTODO,
        payload: {...todo, id: id++},
    };
}
```